akka-docs/rst/scala/cluster-sharding.rst

```diff
@@ -262,8 +262,7 @@ Note that stopped entities will be started again when a new message is targeted
 Graceful Shutdown
 -----------------
 
-You can send the message ``ClusterSharding.GracefulShutdown`` message (``ClusterSharding.gracefulShutdownInstance
-in Java) to the ``ShardRegion`` actor to handoff all shards that are hosted by that ``ShardRegion`` and then the
```

Comment:
I want to remove stuff about Java API from the Scala API docs.
This probably originates from the fact that sharding originally had 1 documentation page for both APIs :)